### PR TITLE
Switch org revamp fix

### PR DESCRIPTION
**Pull request by yksolanki9**
Changes:
- Hide active org card when user logs in
- Make searchbar sticky only in android
- Show default org at top of list
- Make active org card clickable

https://www.loom.com/share/62c3d86196ac40829427d42272c48d8a

### CODE REVIEW

**Review by iravisankar — APPROVED** [review]

File: src/app/auth/switch-org/switch-org.page.html [hunk 1]
```diff
@@ -19,24 +19,32 @@
...
-      <ng-container *ngIf="primaryOrg$ | async as primaryOrg">
-        <app-active-org-card [org]="activeOrg" [isPrimaryOrg]="activeOrg.id === primaryOrg.id"></app-active-org-card>
+  <ng-container *ngIf="navigateBack">
+    <ng-container *ngIf="!isLoading; else skeletonActiveOrg">
+      <ng-container *ngIf="activeOrg$ | async as activeOrg">
+        <ng-container *ngIf="primaryOrg$ | async as primaryOrg">
+          <app-active-org-card
+            [org]="activeOrg"
+            [isPrimaryOrg]="activeOrg.id === primaryOrg.id"
+            (orgSelected)="switchToOrg(activeOrg)"
```
Comment (Dimple16) [inline src/app/auth/switch-org/switch-org.page.html:29]: `switchOrg()`?

**Review by Dimple16 — APPROVED** [review]

File: src/app/auth/switch-org/switch-org.page.html (hunk 1 above)
Comment (yksolanki9) [reply src/app/auth/switch-org/switch-org.page.html]: Done

File: src/app/auth/switch-org/switch-org.page.ts [hunk 2]
```diff
@@ -67,13 +71,16 @@ export class SwitchOrgPage implements OnInit, AfterViewChecked {
   }
 
   ionViewWillEnter() {
+    if (!this.platform.is('ios')) {
+      this.searchContainerRef.nativeElement.classList.add('switch-org__content-container--sticky');
```
Comment (Julias0) [inline src/app/auth/switch-org/switch-org.page.ts:75]: please use ngClass for things like these

File: src/app/auth/switch-org/switch-org.page.ts [hunk 3]
```diff
@@ -93,24 +100,27 @@ export class SwitchOrgPage implements OnInit, AfterViewChecked {
...
     this.primaryOrg$ = this.offlineService.getPrimaryOrg();
 
-    this.activeOrg$.subscribe(noop);
-    this.primaryOrg$.subscribe(noop);
-
-    const currentOrgs$ = forkJoin([this.orgs$, this.activeOrg$]).pipe(
-      switchMap(([orgs, activeOrg]) => of(orgs.filter((org) => org.id !== activeOrg.id)))
+    const currentOrgs$ = forkJoin([this.orgs$, this.primaryOrg$, this.activeOrg$]).pipe(
+      map(([orgs, primaryOrg, activeOrg]) => {
+        const currentOrgs = [primaryOrg, ...orgs.filter((org) => org.id !== primaryOrg.id)];
```
Comment (Julias0) [inline src/app/auth/switch-org/switch-org.page.ts:107]: wouldnt orgs always have primaruy org?

File: src/app/auth/switch-org/switch-org.page.ts [hunk 4]
```diff
@@ -93,24 +100,27 @@ export class SwitchOrgPage implements OnInit, AfterViewChecked {
...
+        if (this.navigateBack) {
```
Comment (Julias0) [inline src/app/auth/switch-org/switch-org.page.ts:108]: Didnt understand this

File: src/app/auth/switch-org/switch-org.page.ts (hunk 3 above)
Comment (yksolanki9) [reply src/app/auth/switch-org/switch-org.page.ts:107]: This is done to keep primary org on the top of the orgs list

File: src/app/auth/switch-org/switch-org.page.ts (hunk 4 above)
Comment (yksolanki9) [reply src/app/auth/switch-org/switch-org.page.ts:108]: This is done to keep primary org on the top of the orgs list

File: src/app/auth/switch-org/switch-org.page.ts (hunk 2 above)
Comment (yksolanki9) [reply src/app/auth/switch-org/switch-org.page.ts]: Done

**Review by Julias0 — APPROVED** [review]

**Pull request merged**